docs/wiki_ja/project_structure.md

```
@@ -22,52 +22,47 @@ Polimoneyプロジェクトは、複数のコアシステムと関連ファイ
-   - GitHub Pagesへのデプロイ用のGitHub Actionsワークフロー
-   - 継続的インテグレーション（CI）とデプロイ（CD）の自動化
-   - 静的サイトホスティングの最適化
+  - Vercelでのホスティングを制御する`vercel.json`
+  - Next.js のサーバーレス実行を前提としたデプロイ手順
+  - 必要に応じた手動デプロイ／プレビュー運用
 
 ## プロジェクト構造図
 
+> フロントエンドの Next.js ソースは `frontend/` ディレクトリ以下に集約されている。
+
 ```
```

markdownlint-cli2 (0.20.0)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)